README.md

```diff
@@ -1,18 +1,23 @@
-# Mbin
+<p align="center">
+    <img src="docs/images/mbin.png" alt="Mbin logo" width="400">
+</p>
+<p align="center">
+  <a href="https://github.com/MbinOrg/mbin/actions/workflows/action.yaml?query=branch%3Amain"><img src="https://github.com/MbinOrg/mbin/actions/workflows/action.yaml/badge.svg?branch=main" alt="GitHub Actions Workflow"></a>
+  <a href="https://github.com/MbinOrg/mbin/actions/workflows/psalm.yml?query=branch%3Amain"><img src="https://github.com/MbinOrg/mbin/actions/workflows/psalm.yml/badge.svg?branch=main" alt="Psalm Security Scan"></a>
+  <a href="https://hosted.weblate.org/engage/mbin/"><img src="https://hosted.weblate.org/widgets/mbin/-/svg-badge.svg" alt="Translation status"></a>
+  <a href="https://matrix.to/#/#mbin:melroy.org"><img src="https://img.shields.io/badge/chat-on%20matrix-brightgreen" alt="Matrix chat"></a>
+  <a href="https://github.com/MbinOrg/mbin/blob/main/LICENSE"><img src="https://img.shields.io/badge/AGPL%203.0-license-blue" alt="License"></a>
+</p>
 
-Mbin is a fork of [/kbin](https://codeberg.org/Kbin/kbin-core), community-focused. Feel free to discuss on [Matrix](https://matrix.to/#/#mbin:melroy.org) and to create Pull Requests.
+## Introduction
 
-> [!Important]
-> Mbin is focused on what the community wants, pull requests can be merged by any repo maintainer (with merge rights in GitHub). Discussions take place on [Matrix](https://matrix.to/#/#mbin:melroy.org) then _consensus_ has to be reached by the community. If approved by the community, only one approval on the PR is required by one of the Mbin maintainers. It's built entirely on trust.
+Mbin is a decentralized content aggregator, voting, discussion and microblogging platform running on the fediverse. It can
+communicate with various ActivityPub services, including but not limited to: Mastodon, Lemmy, Pixelfed, Pleroma and PeerTube.
 
-Mbin is a decentralized content aggregator, voting, discussion and microblogging platform running on the fediverse network. It can
-communicate with many other ActivityPub services, including Kbin, Mastodon, Lemmy, Pleroma, Peertube. The initiative aims to
-promote a free and open internet.
+Mbin is a fork and continuation of [/kbin](https://codeberg.org/Kbin/kbin-core), community-focused. Feel free to discuss on [Matrix](https://matrix.to/#/#mbin:melroy.org) and pull requests are always welcome.
```

Comment:
```suggestion
Mbin is a fork and continuation of [/kbin](https://codeberg.org/Kbin/kbin-core), but community-focused. Feel free to chat on [Matrix](https://matrix.to/#/#mbin:melroy.org). Pull requests are always welcome.
```

Not sure if the later sentence was meant to indicate that you should discuss pull requests on Matrix (in which case, I can update the sentence to reflect), or if there are supposed to be two separate ideas (you can chat on Matrix, and separately, pull requests are welcome).